tests/common_fixtures.py

```diff
@@ -501,7 +502,8 @@ def rundb_mock() -> RunDBMock:
     config.dbpath = "http://localhost:12345"
 
     # Create the default project to mimic real MLRun DB (the default project is always available for use):
-    mlrun.get_or_create_project("default")
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        mlrun.get_or_create_project("default", context=tmp_dir)
 
     yield mock_object
```

Comment:
Should be within the context manager scope (otherwise at this point the tmp dir is already deleted).